app.py

```diff
@@ -144,12 +144,14 @@ def setGridBoundries():
             'display_name': result.get('display_name', 'Name not set'),
             'mission_start_time': str(result['mission_start_time'])
         }
+        field_features = data['field_features']
 
         response_body = {
             'status': 'success',
             # 'grids': grids,
             'flight_details': flight_details,
-            'features': features
+            'features': features,
+            'field_features': field_features
```

Comment:
is this being used anywhere?

Reply:
This is used to get the data that is filled in the 'Add Field Features' like the crop type and lead scientist